src/Microsoft.DotNet.Interactive.Formatting.Tests/HtmlFormatterTests.cs

```diff
@@ -305,7 +306,21 @@ is a
                           $"{PlainTextBegin}{instance.HtmlEncode()}{PlainTextEnd}");
             }
 
+            [Fact]
+            public void It_does_not_affect_BigInteger()
+            {
+                var formatter = HtmlFormatter.GetPreferredFormatterFor(typeof(BigInteger));
+
+                var writer = new StringWriter();
+
+                var instance = BigInteger.Parse("78923589327589332402359");
 
+                formatter.Format(instance, writer);
+
+                writer.ToString()
+                    .Should()
+                    .Be("<div class=\"dni-plaintext\">78923589327589332402359</div>");
```

Comment:
Current `main` would output a table (like in the screenshot). We want to see a plain text instead